### ISSUE TITLE
LoadError: UndefVarError: @osx_only not defined

### ISSUE THREAD

**Issue by femtotrader** [bicycle1885/Snappy.jl#6]
With Julia 0.6 under Mac OS 10.12.5

```julia
julia> Pkg.build("Snappy")
INFO: Building Homebrew
INFO: Building Snappy
WARNING: BinDeps.shlib_ext is deprecated.
  likely near /Users/femto/.julia/v0.6/Snappy/deps/build.jl:13
========================================================================[ ERROR: Snappy ]=========================================================================

LoadError: UndefVarError: @osx_only not defined
while loading /Users/femto/.julia/v0.6/Snappy/deps/build.jl, in expression starting on line 15

==================================================================================================================================================================

=========================================================================[ BUILD ERRORS ]=========================================================================

WARNING: Snappy had build errors.

 - packages with build errors remain installed in /Users/femto/.julia/v0.6
 - build the package(s) and all dependencies with `Pkg.build("Snappy")`
 - build a single package by running its `deps/build.jl` script

==================================================================================================================================================================
```

**Comment by femtotrader** [comment]
See also this Travis build (Linux - Julia 0.6)
https://travis-ci.org/bicycle1885/Snappy.jl/jobs/252430909

**Comment by femtotrader** [comment]
Required by https://github.com/JuliaComputing/Parquet.jl/issues/6

**Comment by dfdx** [comment]
Created a PR for this: 
https://github.com/bicycle1885/Snappy.jl/pull/8

**Comment by bicycle1885** [comment]
This is fixed in #8 by @dfdx.

**Comment by femtotrader** [comment]
Thanks @dfdx and @bicycle1885
